context-free.md

```diff
@@ -1,22 +1,23 @@
 # Context Free
 
 ##  Who is this programming language for?
-
+This programming language is for anyone interested in art and context free grammars.
 
 ## What is easy to do in this language? Why is it easy?
-
+It is easy to generate recuring shapes.
+This is easy because the language has an intuitive form for recursion.
 
 ## What is hard to do in this language? Why is it hard?
-
+It is difficult to write out words and letters because the programmer has to figure out how each of the letters come together and then implement the shape. 
 
 ## How did you learn how to program in this language?
-
+I learned how to program in this language by examining the welcome code, reading through a few examples and trying out some simple shapes on my own.
```

Comment:
I did the same. After I did the assignment I found that Prof. Ben had posted [this link](https://web.archive.org/web/20100620070452/http://contextfreeart.org/mediawiki/index.php/CFDG_HOWTO) on Piazza. But I've found reading examples of programs to be the best way to learn a new language for me anyway.